### DISCUSSION TITLE
µRVL-DD possibility

### DISCUSSION THREAD

**Discussion by AdansWorkshop** [Aurelio92/RVLoader#51] [Ideas]
https://www.digikey.com/en/products/detail/lattice-semiconductor-corporation/ICE40LP384-CM49/3974677
![image](https://github.com/Aurelio92/RVLoader/assets/74674130/a41a8731-de55-403c-97e8-7c80b167285e)
The current RVL-DD uses an FPGA from Lattice Semiconductor. The ICE series FPGAs are far smaller and cheaper and could be used in the development of a µRVL-DD (for those who don't know, the greek letter µ is a shorthand for the prefix micro). It could bring _even smaller_ functional Wiis to the modding community.

**Comment by Aurelio92** [comment]
As I have already explained on Discord, this cannot work. Also this has nothing to do with RVLoader, so I am closing this discussion.
